helmfile.d/values/opensearch/configurer.yaml.gotmpl

```diff
@@ -35,7 +35,16 @@ config:
     rolloverAge: {{ .Values.opensearch.ism.rolloverAgeDays }}
 
   {{- if .Values.opensearch.snapshot.enabled }}
-  snapshotRepository: {{ .Values.opensearch.snapshot.repository }}
+  snapshots:
+    repository: {{ .Values.opensearch.snapshot.repository }}
+    min: {{ .Values.opensearch.snapshot.min }}
+    max: {{ .Values.opensearch.snapshot.max }}
+    backupSchedule: {{ .Values.opensearch.snapshot.backupSchedule | quote }}
+    retentionSchedule: {{ .Values.opensearch.snapshot.retentionSchedule | quote }}
+    retentionAge: {{ .Values.opensearch.snapshot.retentionAge }}
+    # Ignore security plugin index as per
+    # https://opendistro.github.io/for-elasticsearch-docs/docs/elasticsearch/snapshot-restore/#security-plugin-considerations
```

Comment:
I think we might want to update the link here to point to [opensearch](https://opensearch.org/docs/latest/api-reference/snapshots/restore-snapshot/).
They don't mention anything about the security plugin considerations in the opensearch docs however.